bluesky/callbacks/core.py

```diff
@@ -1,7 +1,8 @@
 """
 Useful callbacks for the Run Engine
 """
-from itertools import count
```

Comment:
changed because the word 'count' usually means `bluesky.plans.count` in this codebase